test/unit/rules/no-implicit-this-test.js

```diff
@@ -29,42 +29,11 @@ let good = [
     template: '{{foo-bar data-test-foo}}',
   },
   {
-    template: `import { hbs } from 'ember-cli-htmlbars';
-      import { setComponentTemplate } from '@ember/component';
-      import templateOnly from '@ember/component/template-only';
-      export const SomeComponent = setComponentTemplate(hbs\`{{book}}\`, templateOnly());`,
+    template: '<template>{{book}}</template>',
     meta: {
       filePath: 'layout.gjs',
     },
   },
-  {
-    template: `import { hbs } from 'ember-cli-htmlbars';
-      import { setComponentTemplate } from '@ember/component';
-      import templateOnly from '@ember/component/template-only';
-      export const SomeComponent = setComponentTemplate(hbs\`{{book}}\`, templateOnly());`,
```

Comment:
Those tests were also asserting the completely wrong thing: the are not strict mode, so _should_ trigger a linting error. Moved them to `bad`